cdc/cdc/owner/changefeed.go

```diff
@@ -73,38 +71,43 @@ type changefeed struct {
 	metricsChangefeedResolvedTsGauge      prometheus.Gauge
 	metricsChangefeedResolvedTsLagGauge   prometheus.Gauge
 
-	newDDLPuller func(ctx cdcContext.Context, startTs uint64) (DDLPuller, error)
-	newSink      func() DDLSink
+	// newDDLPuller func(ctx cdcContext.Context, startTs uint64) (DDLPuller, error)
+	// newSink      func() DDLSink
 	newScheduler func(ctx cdcContext.Context, startTs uint64) (scheduler, error)
 }
 
 func newChangefeed(id model.ChangeFeedID, gcManager gc.Manager) *changefeed {
 	c := &changefeed{
 		id: id,
 		// The scheduler will be created lazily.
-		scheduler:        nil,
-		barriers:         newBarriers(),
+		scheduler: nil,
+		// barriers:         newBarriers(),
 		feedStateManager: new(feedStateManager),
 		gcManager:        gcManager,
 
 		errCh:  make(chan error, defaultErrChSize),
 		cancel: func() {},
 
-		newDDLPuller: newDDLPuller,
-		newSink:      newDDLSink,
+		// newDDLPuller: newDDLPuller,
+		// newSink:      newDDLSink,
 	}
 	c.newScheduler = newScheduler
 	return c
 }
 
+// TODO: modify for tikv cdc
 func newChangefeed4Test(
 	id model.ChangeFeedID, gcManager gc.Manager,
-	newDDLPuller func(ctx cdcContext.Context, startTs uint64) (DDLPuller, error),
-	newSink func() DDLSink,
+	/*
+		newDDLPuller func(ctx cdcContext.Context, startTs uint64) (DDLPuller, error),
+		newSink func() DDLSink,
+	*/
 ) *changefeed {
 	c := newChangefeed(id, gcManager)
-	c.newDDLPuller = newDDLPuller
-	c.newSink = newSink
+	/*
```

Comment:
can you directly remove these codes?

Reply:
> can you directly remove these codes?

The way to comment the code is for the convenience of review, I will delete these comments later.

Reply:
ok